src/prpy/planning/chomp.py

```diff
@@ -84,7 +85,20 @@ def sync(self, robot):
                         body_name, os.path.basename(cache_path)
                     )
 
-                    self.module.computedistancefield(body, cache_filename=cache_path)
+                    # Temporarily disable other bodies for distance field computation
+                    other_bodies = []
+                    for other_body in self.env.GetBodies():
+                        if other_body == body:
+                            continue
+                        if not other_body.IsEnabled():
```

Comment:
What does `body.IsEnabled()` return if the body has a mixture of enabled and disabled links? It might be safer to remove this check and disable everything.